extensions/azurePublish/src/services/cosmosDBService.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { CosmosDBManagementClient } from '@azure/arm-cosmosdb';
+import { TokenCredentials } from '@azure/ms-rest-js';
+
+import { CosmosDBConfig } from '../azureResourceManager/azureResourceManagerConfig';
+
+import { throwNotImplementedError } from './throwNotImplementedError';
+
+export const createCosmosDBService = (token: string, subscriptionId: string) => {
```

Comment:
Nit:  We should be consistent in naming the service creation methods.  The LUIS one has Provisioning in the name, but this Cosmos one does not.